src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -280,6 +281,62 @@ def test_aks_create_with_ingress_appgw_addon(self, resource_group, resource_grou
                 'addonProfiles.ingressApplicationGateway.config.subnetCIDR', "10.232.0.0/16")
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_with_namespace_enabled(self, resource_group, resource_group_location="westus2"):
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name,
+            'ssh_key_value': self.generate_ssh_keys()
+        })
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-namespace-resources --ssh-key-value={ssh_key_value}'
+        self.cmd(create_cmd, checks=[
+            self.check('provisioningState', 'Succeeded'),
+            self.check('enableNamespaceResources', True)
+        ])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_update_enable_namespace(self, resource_group, resource_group_location="westus2"):
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name,
+            'ssh_key_value': self.generate_ssh_keys()
+        })
+        
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --ssh-key-value={ssh_key_value}'
+        self.cmd(create_cmd, checks=[
+            self.check('provisioningState', 'Succeeded'),
+        ])
+
+        update_cmd = 'aks update --resource-group={resource_group} --name={name} --enable-namespace-resources'
+        self.cmd(update_cmd, checks=[
+            self.check('enableNamespaceResources', True)
+        ])
+
+        update_cmd = 'aks update --resource-group={resource_group} --name={name} --disable-namespace-resources'
+        self.cmd(update_cmd, checks=[
+            self.check('enableNamespaceResources', False)
+        ])
+
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_get_credentials_at_namespace_scope(self, resource_group):
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name,
+            'ssh_key_value': self.generate_ssh_keys()
+        })
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-namespace-resources --ssh-key-value={ssh_key_value}'
+        self.cmd(create_cmd)
+        print("Cluster created, sleeping for 60 seconds")
+        sleep(60) # Sleep for 60 seconds to allow hydration of namespaces
```

Comment:
Please use `time.sleep(60)` instead of `sleep(60)`, for the context please refer to https://github.com/Azure/azure-cli/pull/21880

Reply:
We don't want to mock out the sleep for this test. There is sometimes a delay in hydrating resources, if we don't sleep, the test will fail.

Reply:
Updated to use time.sleep instead of sleep.